src/js/components/Button/propTypes.js

```diff
@@ -68,7 +68,7 @@ if (process.env.NODE_ENV !== 'production') {
     primary: PropTypes.bool,
     reverse: PropTypes.bool,
     secondary: PropTypes.bool,
-    size: PropTypes.oneOf(['small', 'medium', 'large']),
+    size: PropTypes.oneOf(['small', 'medium', 'large', 'xlarge']),
```

Comment:
See `index.d.ts` comment.

```suggestion
    size: PropTypes.oneOfType([
      PropTypes.oneOf(['small', 'medium', 'large']),
      PropTypes.string,
    ]),
```